packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1247,12 +1247,27 @@ class RenderEditable extends RenderBox {
   /// programmatically manipulate its `value` or `selection` directly.
   /// {@endtemplate}
   void selectPosition({ @required SelectionChangedCause cause }) {
+    selectPositionAt(from: _lastTapDownPosition, cause: cause);
+  }
+
+  /// Select text between the global positions [from] and [to].
+  void selectPositionAt({ @required Offset from, Offset to, @required SelectionChangedCause cause }) {
     assert(cause != null);
```

Comment:
Since "from" is required, is from = null ok? If not, assert?

Reply:
Thanks. Added here and below.